src/main/java/org/opentripplanner/updater/trip/TimetableSnapshotSource.java

```diff
@@ -350,6 +354,14 @@ public UpdateResult applyTripUpdates(
           failuresByRelationship
         );
       }
+
+      var warnings = Multimaps.index(updateResult.warnings(), w -> w);
+      warnings
+        .keySet()
+        .forEach(key -> {
+          var count = warnings.get(key).size();
+          LOG.info("[feedId: {}] {} warnings of type {}", feedId, count, key);
+        });
```

Comment:
Could we move the logging into a separate method?

Reply:
Done.